### PR TITLE
ui: enlarge search box on dashboard

### LINKED ISSUE #8812: Search input box is too small. It should be fully flexible since there is no boundary (border) on this `input` box.
Search input box is too small. It should be fully flexible since there is no boundary (border) on this `input` box.

Screen Shot 2021-04-12 at 12 35 17

_Originally posted by @tnir in https://github.com/gitpod-io/gitpod/issues/3727#issuecomment-817455895_

### PR DESCRIPTION
**Pull request by tnir**
Signed-off-by: Takuya Noguchi <takninnovationresearch@gmail.com>

## Description
Enlarges the search box on dashboard.

See the (simplified) sample code:

https://codepen.io/tnir/pen/GRyJoRE

## Related Issue(s)
Fixes #8812 (part of #3727)

Improves #3295

## How to test
Prepare an environment and resize the window size responsively.

## Release Notes
```release-note
NONE
```
(or `Enlarge search box on dashboard`)

## Documentation
No.

### CODE REVIEW

**Comment by roboquat** [comment]
@tnir: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by tnir** [comment]
@gtsiolis What do you think?

File: components/dashboard/src/workspaces/Workspaces.tsx [hunk 1]
```diff
@@ -72,8 +72,6 @@ export default function () {
                                     }}
                                 />
                             </div>
-                            <div className="flex-1" />
-                            <div className="py-3"></div>
```
Comment (tnir) [inline components/dashboard/src/workspaces/Workspaces.tsx:76]: Tackles a technical debt from #5555

**Comment by tnir** [comment]
@svenefftinge Could you review this?

**Comment by easyCZ** [comment]
@tnir Thank you for your contribution!

@gtsiolis Would you be able to take a look? This looks to fall in your area of expertise.

**Comment by gtsiolis** [comment]
Looking at this now! 👀 

/werft run

**Comment by gtsiolis** [comment]
Thanks for contributing, @tnir! 🧡 

You'll also need to sign a Contributor License Agreement (CLA)[[1](https://en.wikipedia.org/wiki/Contributor_License_Agreement)] once before merging your first contribution. Looping in @meysholdt to reach out about the CLA. 🏓

In the meatime, please bare with us while we debug why the build is failing for this. 🙏

**Comment by tnir** [comment]
Thanks @gtsiolis. I have not been sure why https://www.gitpod.io/cla (originated from https://github.com/gitpod-io/website/blob/45ad1e610c189c3c36585372ccc5a4bf32d37a07/gitpod/docs/contribute/features-and-patches/index.md?plain=1#L16) is dead... 😭

**Comment by gtsiolis** [comment]
> I have not been sure why https://www.gitpod.io/cla ...

@tnir That's currently a private repository where some team members experiment with a CLA automation process so that it's a self-serve process rather than manually reaching out[[1](https://github.com/gitpod-io/gitpod/pull/8813#issuecomment-1072369406)] to contributors.

[FWIW](https://en.wiktionary.org/wiki/FWIW), that link to the private repository was added back in https://github.com/gitpod-io/website/pull/1187. I've opened https://github.com/gitpod-io/website/pull/1789 to remove this temporarily until we make the repository public if needed.

**Comment by gtsiolis** [comment]
One more time.

/werft run

**Comment by gtsiolis** [comment]
Build succedeed! Triggering one more time because of the timeout.

/werft run

**Comment by tnir** [comment]
@meysholdt When can I get reached out from you? 🙏

**Comment by tnir** [comment]
Ping @meysholdt

**Comment by gtsiolis** [comment]
@tnir Sorry this is taking longer than usual but please bare with us as most of the company is currently at our offsite. I promise to get back to this next week. 🎗️

**Review by easyCZ — APPROVED** [review]

**Comment by easyCZ** [comment]
Looks like I've accidentally caused the PR to be merged by approving it (did not notice there's no /hold). Sorry, that was not the intent, just intended to indicate the changes look good. 

We'll discuss internally if we need to revert. Sorry about this.

**Comment by easyCZ** [comment]
I've raised a revert for this in https://github.com/gitpod-io/gitpod/pull/8813. Unfortunately, to get this change in @tnir will need to raise a new PR (such that it's attributed to you). Sorry for the inconvenience.

I'm also looking into how to add hold to PRs by default to prevent this default case scenario.

**Comment by jankeromnes** [comment]
Thanks again @tnir for your great contribution!

FYI, for convenience, I've restored your commit into a new Pull Request: https://github.com/gitpod-io/gitpod/pull/8994 -- the commit is still attributed to you, but if you prefer, please feel free to open it in another Pull Request, and I'd be happy to approve it.

Will also ping internally to see what we can do to accelerate the CLA process. 🙂

**Comment by tnir** [comment]
Thanks @jankeromnes. Can you help me accelerate the CLA process at https://github.com/gitpod-io/gitpod/issues/7864#issuecomment-1088323387

> Will also ping internally to see what we can do to accelerate the CLA process. 🙂

**Pull request merged**